libjpeg-turbo.lua

```diff
@@ -0,0 +1,606 @@
+project "jpeg"
+
+  local prj = project()
+  local prjDir = prj.basedir
+
+  -- -------------------------------------------------------------
+  -- project
+  -- -------------------------------------------------------------
+
+  -- common project settings
+
+  dofile (_BUILD_DIR .. "/3rdparty_static_project.lua")
+
+  -- project specific settings
+
+  uuid "9A746419-F4A6-4D7A-8819-689F1F12C3AF"
+
+  flags {
+    "NoPCH",
+  }
+
+  files {
+    "jaricom.c",
+    "jcapimin.c",
+    "jcapistd.c",
+    "jcarith.c",
+    "jccoefct.c",
+    "jccolor.c",
+    "jcdctmgr.c",
+    "jchuff.c",
+    "jcinit.c",
+    "jcmainct.c",
+    "jcmarker.c",
+    "jcmaster.c",
+    "jcomapi.c",
+    "jcparam.c",
+    "jcphuff.c",
+    "jcprepct.c",
+    "jcsample.c",
+    "jctrans.c",
+    "jdapimin.c",
+    "jdapistd.c",
+    "jdarith.c",
+    "jdatadst.c",
+    "jdatasrc.c",
+    "jdcoefct.c",
+    "jdcolor.c",
+    "jddctmgr.c",
+    "jdhuff.c",
+    "jdinput.c",
+    "jdmainct.c",
+    "jdmarker.c",
+    "jdmaster.c",
+    "jdmerge.c",
+    "jdphuff.c",
+    "jdpostct.c",
+    "jdsample.c",
+    "jdtrans.c",
+    "jerror.c",
+    "jfdctflt.c",
+    "jfdctfst.c",
+    "jfdctint.c",
+    "jidctflt.c",
+    "jidctfst.c",
+    "jidctint.c",
+    "jidctred.c",
+    "jmemmgr.c",
+    "jmemnobs.c",
+    "jquant1.c",
+    "jquant2.c",
+    "jutils.c",
+  }
+
+  includedirs {
+    "./",
+  }
+
+  -- -------------------------------------------------------------
+  -- configurations
+  -- -------------------------------------------------------------
+
+  if (os.is("windows")) then
+
+    defines {
+      "TURBO_FOR_WINDOWS",
+    }
+
+    files {
+      "jsimd_none.c",
```

Comment:
Can the jsimd files be put into a table above so they can easily be updated in the future.  Like https://devtopia.esri.com/3rdparty/skia/blob/runtimecore/skia.lua#L344